samples/bluetooth/peripheral_power_profiling/prj.conf

```diff
@@ -31,5 +31,10 @@ CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
 
 CONFIG_POWEROFF=y
+CONFIG_PM_DEVICE=y
 
 CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_ASSERT=n
+CONFIG_BT_ASSERT=n
+CONFIG_ASSERT_VERBOSE=n
```

Comment:
Should we consider disabling `CONFIG_NCS_SAMPLES_DEFAULTS`?

Reply:
> Should we consider disabling `CONFIG_NCS_SAMPLES_DEFAULTS`?

Yes, I think we should disable `CONFIG_NCS_SAMPLES_DEFAULTS`.

Reply:
> Why you are disabling asserts?

I disabled asserts to reduce code and power consumption.